src/transformers/modeling_tf_bert.py

```diff
@@ -936,7 +937,9 @@ def call(
 class TFBertLMHeadModel(TFBertPreTrainedModel, TFCausalLanguageModelingLoss):
     def __init__(self, config, *inputs, **kwargs):
         super().__init__(config, *inputs, **kwargs)
-        assert config.is_decoder, "If you want to use `TFBertLMHeadModel` as a standalone, add `is_decoder=True.`"
+
+        if not config.is_decoder:
+            logger.info("If you want to use `TFBertLMHeadModel` as a standalone, add `is_decoder=True.`")
```

Comment:
@patrickvonplaten do we really need an assert rather than a warning? This model is untested in the commontests right now because of this

Reply:
I see! -> yeah a warning would be fine as well

Reply:
great! I think the same applies to `modeling_bert.py` I guess. Maybe change it there as well?